docs/quick_start.rst

```diff
@@ -21,3 +21,38 @@ Output:
  * output.nwb: NWB file including spike times
  * log.txt: run log
  * qc_figs: index.html includes cell figures and feature table and sweep.html includes sweep figures
+
+
+Step-by-step guideline
```

Comment:
I think having a separate "Step-by-step" section is confusing. We should include the guideline inside the existing quick start section. 
Something like:
To get started, install IPFX via pip into a fresh conda evironment. For details see the Installation page (link to the installation page)
Next, download some data files  according to the instructions in the Data Access (link) page.
To process a dataset saved in the nwb file, run:
...

Reply:
modified as suggested, thanks.